src/internal/m365/collection/drive/permission.go

```diff
@@ -390,7 +390,12 @@ func UpdateLinkShares(
 		}
 
 		if err != nil {
-			el.AddRecoverable(ctx, clues.Stack(err))
+			if graph.IsErrSharingDisabled(err) {
+				logger.Ctx(ctx).Info("unable to restore link share; sharing disabled")
```

Comment:
Should we add an alert as well, like [here](https://github.com/alcionai/corso/blob/e46119a81878304b2167bd36da549186e7186f5e/src/pkg/services/m365/api/mail.go#L441)? Also at line 510

Reply:
IMO, I don't think alert makes sense in this case as the sharing was disabled by the user. Even if we do alert, I don't think we should do it for every single item.

Reply:
I agree with Abin.  Alerts should be reserved for cases where we think the system has unintended consequences/conditions.  Abiding user configurations shouldn't cascade to user visible attention.